### PR TITLE
[WIP]: Create supporters page

### LINKED ISSUE #49: Supporters page
There are a few companies that sponsor our project or on which free services it depends. I think we should have a page to acknowledge that. I'll start a list but I'm sure I'm forgetting some, so please add to it if you see the need.
- While the GitHub currently is still on a paid plan by Diaspora Inc. I see no reason to keep it that way. Once we do not pay anything for their service we should acknowledge it.
- The company that sponsored the wildcard cert for diasporafoundation.org <del>that I can't remember the name of... (/cc @denschub ;) )</del>
- WebTranslateIt
- Loomio
- @denschub for paying the server running everything on diasporafoundation.org
- Pivotal for their past support

### PR DESCRIPTION
**Pull request by goobertron**
Will fix #49 when completed.

@denschub @SuperTux88 @svbergerem @Flaburgan am I going in the right direction with this? Layout, content, etc.

Let's discuss that before I go any further.

### CODE REVIEW

File: app/views/pages/supporters.html.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<% content_for :wrapper_class, 'supporters-page' %>
+
+<div class="hero" id="masthead">
+  <div class="container">
+    <h1><%= t '.menu_title' %></h1>
+    <h2 class="lead"><%= t '.byline' %></h2>
+  </div>
+</div>
+
+<div class="page-header">
```
Comment (Flaburgan) [inline app/views/pages/supporters.html.erb]: This should probably be in the container, it feels weird to see it taking the whole page width.

**Review by Flaburgan — CHANGES_REQUESTED** [review]
It looks pretty nice! Where do you plan to place a link to this page? In the footer?

File: app/views/pages/supporters.html.erb (hunk 1 above)
Comment (svbergerem) [inline app/views/pages/supporters.html.erb]: I agree.

**Comment by denschub** [comment]
> am I going in the right direction with this?

Sure, why not! Not too sure, however, if we still should include Pivotal and Kickstarter. Those days are over and sometimes, you have to get rid of old stories.

**Comment by Flaburgan** [comment]
@goobertron do you still want to work on that? I now have my dev install working again so I can review your work once you fixed the above remarks ;)

Btw, should we include bountysource there?

**Comment by goobertron** [comment]
Hi, thanks for prompting me. I had hit another problem with running the dev environment, but I've just tried bundle again and this time it compiled and is working once again (hurrah!). I've placed the strapline inside the container as requested, but there's little more I can do until we've decided what content to include there. At the moment it's just a basic layout with four supporters (GitHub, Travis, Let's Ecrypt and WebTranslateIt) listed, with a brief description I wrote for each.

I'll push the tiny change with the strapline code moved, anyway.

**Comment by SuperTux88** [comment]
> am I going in the right direction with this?

Yes, I like it. But I agree with @denschub, I would only include organizations which are currently supporting us or which we are using. But maybe we could add bountysource instead?

Also, since you mentioned "**people** and organizations" we could maybe combine it with the contributors page from #108, after we decided [which people to list there](https://github.com/diaspora/diaspora-project-site/pull/108#issuecomment-377989620)?

**Comment by goobertron** [comment]
> I would only include organizations which are currently supporting us or which we are using.

Sure; as I said, this WIP so far is only about concept and layout; I just slung in a few things as 'dummy' entries to show how it would look. We need to discuss what content we include and don't include.

I would like to put forward the suggestion, however, to have a separate section underneath the supporters: 'We would also like to thank the following people and organizations that helped Diaspora in its earlier days' as a way of keeping acknowledgement of previous support without suggesting that they are still actively supporting diaspora\*.

We could definitely include contributors on the same page; they certainly support the project, with time and effort rather than money. And we could mention other types of supporters, such as podmins, although not individually by name.

Let's open a separate issue for a contributors page, as we clearly have a lot to discuss about what form that would take.

**Comment by goobertron** [comment]
So if people can give their suggestions for which supporters should be included on this page, I can then move the PR forward.

Currently, among active supporters, we have just GitHub, Travis, WebTranslateIt and SFLC.

I'm not sure I'd consider Bountysource as a supporter; we pay them for using their platform as they take a 10% cut of any donation. That's not what I consider support.

**Comment by SuperTux88** [comment]
> I'm not sure I'd consider Bountysource as a supporter; we pay them for using their platform as they take a 10% cut of any donation. That's not what I consider support.

OK, was just an idea :)

But I have another one we should add to the list: JetBrains for providing RubyMine licenses.

**Comment by SuperTux88** [comment]
I don't know if the old code still works, but I think we can help you with the code. When you just do the page with static dummy content, it will be easy in the end to replace it with the content from the database.

**Comment by denschub** [comment]
This PR got automatically closed by GitHub after we deleted the `master` branch and replaced the default branch with `production`. I did miss that this change will close all open PRs, sorry for that. Since there has been no activity for a while, I think this is fine, but if someone feels strongly about this, please let me know and we can restore this. :)

**Pull request closed without merging**